client/src/locales/ru.ts

```diff
@@ -215,11 +217,11 @@ export default {
 			"subreddits": "Сабреддиты: {url}",
 		},
 		"messages": {
-			"unknown-status": "Unknown status for add preview response: {status}.",
-			"unknown-error": "An unknown error occurred when getting add preview. Try again later.",
+			"unknown-status": "Неизвестный статус добавления превью: {status}.",
+			"unknown-error": "Произошла неизвестная ошибка при получении превью видео. Попробуйте позже.",
 			"failed-to-get-add-preview":
-				"Failed to get add preview. This is probably a bug, check console for details.",
-			"failed-to-all-videos": "Failed to all videos: {message}",
+				"Не удалось получить превью для видео. Вероятно, это ошибка, подробности в консоли.",
+			"failed-to-all-videos": "Ошибка для всех видео: {message}",
```

Comment:
should I translate lines with errors? If yes, maybe You should come up with an error code for each error? This way it will be easier to understand the meaning of the error in different localizations for You

Reply:
do you mean translate the keys like `failed-to-all-videos`?

Reply:
> do you mean translate the keys like `failed-to-all-videos`?

Yes, strings 220-224, 322-330

Reply:
Those need to stay in english. Those are used to tell the UI what localized string to use.

Reply:
> Those need to stay in english. Those are used to tell the UI what localized string to use.

Ok!